sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -253,7 +254,7 @@ def visit(self, node):
       return self.visit_nested(node)
 
 
-def get_named_nested_pvalues(pvalueish):
+def get_named_nested_pvalues(pvalueish, as_inputs=False):
```

Comment:
In the invocation of `get_named_nested_pvalues()` in `pipeline.py`, do we also need to stringify the tags (so that `None` becomes `'None'`)?

Reply:
This is done elsewhere as needed.